vm/vmExpr.go

```diff
@@ -734,6 +734,27 @@ func invokeExpr(expr ast.Expr, env *Env) (reflect.Value, error) {
 	case *ast.CallExpr:
 		return callExpr(e, env)
 
+	case *ast.DeleteExpr:
+		mapExpr, err := invokeExpr(e.MapExpr, env)
+		if err != nil {
+			return nilValue, newStringError(e, err.Error())
+		}
+
+		keyExpr, err := invokeExpr(e.KeyExpr, env)
+		if err != nil {
+			return nilValue, newStringError(e, err.Error())
+		}
+
```

Comment:
Need to check if Map is nil. Also need to check if map is a pointer to a map.

Reply:
add check id map is nil.

but, if map is a pointer to a map will ok in anko?
if in go:
```go
var xxx = map[string]string{"a": "a"}
delete(&xxx, "a")
```

Does not compile